test/src/e2e/test_datatypes.py

```diff
@@ -0,0 +1,498 @@
+import datetime
+import decimal
+import math
+import uuid
+
+import pytest
+
+from util import pyodbc_connection, create_table, rows_as_values
+
+
+# FIXME: None is converted to an empty string (probably by PyODBC itself?)
```

Comment:
I wonder if it's indeed the PyODBC "feature".
Maybe could try [turbodbc](https://turbodbc.readthedocs.io/en/latest/) in the future.